Grafana 10.3 release

adriens opened the pull request:
[cf https://grafana.com/blog/2024/01/23/grafana-10.3-release-canvas-panel-updates-multi-stack-data-sources-and-more/](https://grafana.com/blog/2024/01/23/grafana-10.3-release-canvas-panel-updates-multi-stack-data-sources-and-more/)

![image](https://github.com/endoflife-date/endoflife.date/assets/5235127/6126ee49-fa0a-431a-a2be-7e921ddc8bdf)


# :point_up:  NBs about `10.3.1`

![image](https://github.com/endoflife-date/endoflife.date/assets/5235127/102ef0f7-4612-4905-8fa3-1909ad2fdf88)

# :bookmark: 

- [Grafana 10.3.1](https://grafana.com/grafana/download/10.3.1/?pg=blog&plcmt=body-txt).
- [Grafana 10.3 release: Canvas panel updates, multi-stack data sources, and more](https://grafana.com/blog/2024/01/23/grafana-10.3-release-canvas-panel-updates-multi-stack-data-sources-and-more/)
- [Tweet](https://twitter.com/grafana/status/1749892885099462956)

adriens committed:
Grafana 10.3 release

cf https://grafana.com/blog/2024/01/23/grafana-10.3-release-canvas-panel-updates-multi-stack-data-sources-and-more/

marcwrobel reviewed (changes requested):
Older versions support and eol dates must be updated according to the product description.

adriens committed:
Update grafana.md

cf https://github.com/endoflife-date/endoflife.date/pull/4511#pullrequestreview-1840032596
